CommonWallet/Classes/Common/Lang/WalletLanguage.swift

```diff
@@ -0,0 +1,39 @@
+/**
+* Copyright Soramitsu Co., Ltd. All Rights Reserved.
+* SPDX-License-Identifier: GPL-3.0
+*/
+
+import Foundation
+
+
+private let kDefaultsKeyName = "l10n_lang"
+
+
+public enum WalletLanguage: String {
+    
+    case english = "en"
+    case khmer = "km"
+
+}
+
+
+public extension WalletLanguage {
+    
+    func save() {
```

Comment:
I think we don't need to store language in user default. Host application either provide one during initialisation or default one based on locale should be used. Current language can be stored by Resolver in memory. What do you think?